build.sbt

```diff
@@ -339,18 +339,3 @@ lazy val reportExtractor = lambda("reportextractor", "reportextractor", Some("co
   .settings(
     riffRaffArtifactResources += (baseDirectory.value / "cfn.yaml", "reportextractor-cfn/cfn.yaml")
   )
-
-lazy val root = (project in file(".")).
```

Comment:
Sorry if I'm being silly, but can you explain why this PR allows us to remove this?

Reply:
It turns out you can completely remove the top level project and sbt will generate it for you so I figured I'd do that. But you're right it's unrelated. I'll revert that part and raise that separately.

I'll start by fixing the build and I'll revert that

Reply:
Happy to include that in this PR if you like. I genuinely didn't understand how it was working!